CustomTabBar/CustomTabBar.swift

```diff
@@ -69,12 +69,24 @@ public protocol CustomTabBarDelegate {
         }
     }
     
+    public var titleFont: UIFont = UIFont.systemFontOfSize(16) {
+        didSet {
+            updateTabs()
+        }
+    }
+    
     @IBInspectable public var lightBackground: Bool = true {
         didSet {
             updateBackground()
         }
     }
     
+    @IBInspectable public var darkBackground: Bool = false {
```

Comment:
So the goal of this is to be able to turn off the visual effect views with blur and just have a simple background color?